tests/security/selinux/fixfiles.pm

```diff
@@ -0,0 +1,58 @@
+# Copyright (C) 2020 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: Test "fixfiles" can fix file SELinux security contexts
+# Maintainer: llzhao <llzhao@suse.com>
+# Tags: poo#65672, tc#1745370
+
+use base "selinuxtest";
```

Comment:
i suggest we create uitls_selinux.pm for store the function create_test_file etc..
then using "use uitils_selinux.pm" instead of "use base xxx", since i do not think the relation is father and child.

Reply:
Thanks for your comments. Your suggestions make sense in general.
But I do not think my "lib/selinuxtest.pm" could became a utility at least ATM though somebody can reuse "create_test_file" etc.
I chose "use base selinuxtest" not only for convenience purpose.
Functions "fixfiles_restore()" and "check_fcontext()" are SELinux related they are designed for SELinux only ATM, so using "base" is proper as it shows father/child on purpose.